string action
string filename
int32 class_label
int32 marker_id

---

bool success
string response
